### PR TITLE
Refactor Create

### PR DESCRIPTION
**Pull request by taoalpha**
This will make it easier to refactor the code for v2ui

### CODE REVIEW

File: code/client/src/pages/Create.jsx [hunk 1]
```diff
@@ -45,108 +44,84 @@ const getGoogleAuthenticatorAppLink = (os) => {
...
+    seed2: generateOtpSeed(),
+    duration: WalletConstants.defaultDuration,
+    doubleOtp: false
+  }).current
 
   const [effectiveTime, setEffectiveTime] = useState()
+  const [otpReady, setOtpReady] = useState(false)
 
-  const generateNewOtpName = () => ONENames.genName(Object.keys(wallets).map(k => wallets[k].name))
-  const [name, setName] = useState(generateNewOtpName())
```
Comment (polymorpher) [inline code/client/src/pages/Create.jsx:67]: Let's keep name. We should allow users to pick their own name and dynamically change the QR code

Comment (taoalpha) [reply code/client/src/pages/Create.jsx:67]: ya but it's not used yet, and we can add it back whenever we need, like in the v2ui. But for now, I want to remove those unused stuff so we can keep it simple for the upgrade

**Comment by polymorpher** [comment]
It looks good, but I think `address` and `walletInfo.doubleOtp` should also be made state variables instead of native variables. Rendering components and functions defined on a per-render basis depend on those. In the case of `address`, being a native variable is harmless because we know by the time `walletInfo.address` is rendered and accessed (when DoneSection is shown), it was already set and won't be changed. But it can be problematic for `doubleOtp`. When its value is changed, components and functions won't be re-rendered and re-computed automatically

**Comment by polymorpher** [comment]
also lots of this error
image

**Comment by polymorpher** [comment]
With two codes, it also goes straight to deployment, rather than waiting for user to confirm (and change spending limit if needed)

**Review by polymorpher — APPROVED** [review]
I made some changes and should have fixed the issues. Do some tests? Especially cases of expert-mode enabled, and scanning is done quickly before worker is done

**Comment by polymorpher** [comment]
fixed it

**Pull request merged**